front/app/containers/Admin/formBuilder/components/FormBuilderToolbox/index.tsx

```diff
@@ -85,10 +97,20 @@ const FormBuilderToolbox = ({
             onClick={() => addAnswer('text')}
           />
           <ToolboxItem
-            icon="info"
+            icon="multiple-choice"
             label={formatMessage(messages.multipleChoice)}
             onClick={() => addAnswer('multiselect')}
           />
+          <ToolboxItem
+            icon="info"
+            label={formatMessage(messages.number)}
+            onClick={() => addAnswer('number')}
+          />
+          <ToolboxItem
+            icon="info"
```

Comment:
Ben provided another icon in the design. Is that icon available, so that we can replace this general "i" icon?

Reply:
@koendehondt Ben sent me the icons, but I haven't had a chance to add them to the component library + publish it yet. Should I add that as a TODO and leave them like this for now? Or make the changes in the component library now?

Reply:
If you have the time today, please do it today. If not, let's get this PR ready for release.

Reply:
I will do it after our meeting :+1:

Reply:
I see the new icons in the UI. 👍